src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -213,8 +214,9 @@ private List<ModificationInfos> execCreateLockoutLine(NetworkStoreListener liste
         Network network = listener.getNetwork();
         String rootReporterId = reporterId + "@" + NETWORK_MODIFICATION_TYPE_REPORT;
         ReporterModel reporter = new ReporterModel(rootReporterId, rootReporterId);
-        String subReportId = "Lockout line " + lineId;
-        Reporter subReporter = reporter.createSubReporter(subReportId, subReportId);
+        StringBuilder taskKey = new StringBuilder();
+        taskKey.append(ModificationType.BRANCH_STATUS.name()).append("_").append(ActionType.LOCKOUT.name());
+        Reporter subReporter = reporter.createSubReporter(taskKey.toString(), "Lockout line ${lineId}", LINE_ID_PARAMETER, lineId);
```

Comment:
I prefer a concise writing for easy reading

  `              Reporter subReporter = reporter.createSubReporter(ModificationType.BRANCH_STATUS + "_" + ActionType.LOCKOUT, "Lockout line ${lineId}", LINE_ID_PARAMETER, lineId);`